<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>693</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="startwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#startwidget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.994, y2:0.971591, stop:0.215909 rgba(0, 124, 162, 255), stop:0.59322 rgba(32, 36, 109, 255), stop:0.960452 rgba(61, 86, 138, 255), stop:1 rgba(61, 86, 138, 255));
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;MV Boli&quot;; color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string>New Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enter">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>520</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:7px;
background-color:rgb(191, 191, 191);
font: 16pt &quot;MV Boli&quot;;color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>520</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:7px;
background-color:rgb(191, 191, 191);
font: 16pt &quot;MV Boli&quot;;color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;MV Boli&quot;; color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string>Please Enter Username:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>410</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
font: 15pt &quot;MV Boli&quot;;color:rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>480</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;MV Boli&quot;; color:;
color: rgb(255, 0, 0);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
